session_he_test.go

```diff
@@ -173,27 +175,78 @@ func TestSessionHE_RatchetDecrypt_CommunicationSkippedMessages(t *testing.T) {
 		d, err = bob.RatchetDecrypt(m0, nil) // Decrypted.
 		require.Nil(t, err)
 		require.Equal(t, []byte("hi"), d)
+
+		d, err = bob.RatchetDecrypt(m4, nil) // Decrypted.
+		require.Nil(t, err)
+		require.Equal(t, []byte("what up bob?"), d)
+
+		d, err = bob.RatchetDecrypt(m5, nil) // Decrypted.
+		require.Nil(t, err)
+		require.Equal(t, []byte("bob?"), d)
+
 	})
 }
 
-func TestSessionHE_SkippedKeysDeletion(t *testing.T) {
+func TestSessionHE_OldKeysDeletion(t *testing.T) {
 	// Arrange.
 	var (
 		bob, _   = NewHE(sk, sharedHka, sharedNhkb, bobPair, WithMaxKeep(2))
 		alice, _ = NewHEWithRemoteKey(sk, sharedHka, sharedNhkb, bobPair.PublicKey(), WithMaxKeep(2))
-		h        = SessionTestHelperHE{t, alice, bob}
 	)
 
 	// Act.
-	m0 := alice.RatchetEncrypt([]byte("Hi"), nil)
+	m0 := alice.RatchetEncrypt([]byte("Hi 1"), nil)
+	m1 := alice.RatchetEncrypt([]byte("Hi 2"), nil)
+	m2 := alice.RatchetEncrypt([]byte("Hi 3"), nil)
+	m3 := alice.RatchetEncrypt([]byte("Hi 4"), nil)
+
+	// Assert.
 
-	h.AliceToBob("Bob!", nil)         // Bob ratchet step 1.
-	h.BobToAlice("Alice?", nil)       // Alice ratchet step 1.
-	h.AliceToBob("How are you?", nil) // Bob ratchet step 2.
+	// This one should be in the db
```

Comment:
I would rather have the comments passed as text to the `require.Nil` function, so that errors can be understood just by looking at the CI errors, instead of needing to immediately refer to the code.